README.md

```diff
@@ -95,3 +119,17 @@ To run examples provided, please make sure these resources are available:
  * Memory: 5.5G
  * Disk space: 20G
 
+## Development notes
+
+Because symbiflow-arch-defs relies on yosys and VPR, it may be useful to
+override the default packaged binaries with locally supplied binaries.  The
+build system allows this via environment variables matching the executable name.
+Here is a list of common environment variables to defined when doing local yosys
+and VPR development.
+
+ - YOSYS : Path to yosys executable to use.
+ - VPR : Path to VPR executable to use.
+ - GENFASM : Path genfasm executable to use.
+
+There are more binaries that are packaged (e.g. VVP), but the packaged versions
+are typically good enough for most use cases.
```

Comment:
Do you need to rerun `cmake` after setting the values?

Reply:
Yes, I'll clarify that.